<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>video_player</class>
 <widget class="QWidget" name="video_player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1314</width>
    <height>906</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>播放影片</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>1190</width>
     <height>848</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
    <item>
     <widget class="QWidget" name="bufferWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>1024</width>
        <height>768</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
      <item>
       <widget class="QLabel" name="startTimeLbl">
        <property name="font">
         <font>
          <family>微軟正黑體</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0:00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="videoSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="EndTimeLbl">
        <property name="font">
         <font>
          <family>微軟正黑體</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0:00</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="playBtn">
        <property name="font">
         <font>
          <family>微軟正黑體</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Play</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pauseBtn">
        <property name="font">
         <font>
          <family>微軟正黑體</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Pause</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
